app/views/SubmitForm.scala.html

```diff
@@ -38,28 +37,34 @@
   nIProtocolEnabled: Boolean)(implicit messages: Messages, appConfig: AppConfig, user: User[_])
 
 @boxRow(divID: String, formID: String, boxTitle: String, boxDescription: String, boxAmount: String, autoCalculated: Boolean = false) = {
-  <div class="grid-row" id=@divID>
-    <div class="column-one-quarter form-hint">@messages(boxTitle)</div>
-    <div class="column-one-half form-hint">@messages(boxDescription)</div>
-        @if(autoCalculated) {
-            <div class="column-one-quarter r-align">
-                <div class="form-group bold-small">
-                    @messages("submit_form.calculatedValue")
-                </div>
-            </div>
-        } else {
-            <div class="column-one-quarter r-align">
-                <div class="form-group">
-                    @moneyInput(
-                        field = form(formID),
-                        label = if (messages.lang.language == "cy") messages(boxAmount) + " " + messages(boxDescription)
-                        else messages(boxTitle) + " " + messages(boxDescription) + " " + messages("confirm_submission.amount"),
-                        decimalPlace = true
-                    )
-                </div>
-            </div>
-        }
-  </div>
+<div class="govuk-summary-list__row" id=@divID>
+    <dt class="govuk-summary-list__key govuk-hint">@messages(boxTitle)</dt>
+    <dd class="govuk-summary-list__value govuk-hint">@messages(boxDescription)</dd>
+    @if(autoCalculated) {
+    <dd class="govuk-summary-list__actions govuk-hint"><strong>
```

Comment:
Can you remove the govuk-hint because it's making the text appear with a grey tint and it's supposed to be black